scraper.py

```diff
@@ -5,31 +5,39 @@
 import csv
 import sys
 import mysql.connector
+from plyer import notification
 
-c = sys.argv[1]
-yr = sys.argv[2]
+c = sys.argv[0]
```

Comment:
this doesn't specify the range of yr and c but specifies the order of the arguments to the file while running it